plugins/aws/acm/acmSingleDomainNameCertificate.js

```diff
@@ -0,0 +1,67 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'ACM Single Domain Name Certificates',
+    category: 'ACM',
+    domain: 'Identity and Access management',
+    description: 'Ensure that ACM single domain name certificates are used instead of wildcard certificates within your AWS account.',
+    more_info: 'ACM certificates are domain validated i.e. the subject field of an ACM certificate identifies a domain name and nothing more. ACM allows you to use an asterisk (*) in the domain name to create an ACM certificate containing a wildcard name that can protect several sites in the same domain.' + 
+        'When you request a wildcard certificate, the asterisk (*) must be in the leftmost position of the domain name and can protect only one subdomain level.',
```

Comment:
```suggestion
    more_info: 'Using wildcard certificates can compromise the security of all sites i.e. domains and subdomains if the private key of a certificate is hacked. So it is recommended to use ACM single domain name certificates instead of wildcard certificates.',
```